README-Setup.md

```diff
@@ -107,6 +107,15 @@ You can override the configuration of the proxy by settings one or more of the f
 |**$SATOSA_SAML2_REQUESTED_ATTRIBUTES**|SAML2 required attributes|name, surname|
 |**$SATOSA_SPID_REQUESTED_ATTRIBUTES**|SPID required attributes|spidCode, name, familyName, fiscalNumber, email|
 
+### PYEUDIW
+
+it is a component designed to implement the EUDI Wallet that aims to create a single secure digital identity and carrying out Relying Party functionalities.
+Pyeudiw is based on the OpenID4VP authentication protocol that allows users to access their digital wallets using a federated identity.
+For more information about EUDI Wallet please visit:
+* the "Italian implementation profile" [page](https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/)
+* the Relying Party authentication documentation [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python)
```

Comment:
```suggestion
### EUDI Wallet 

Satosa-Saml2Spid uses the EUDI Wallet backend based on [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python) also known as pyeudiw.

pyeudiw is based on OpenID4VP and specifically implements an EUDI Wallet Relying Party to provide a Wallet Authentication mechanims within SATOSA.

For more information about EUDI Wallet please visit the "Italian implementation profile" [page](https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/).
```

please move this in https://github.com/italia/Satosa-Saml2Spid/blob/eudi/README-Setup.md, where all the components are listed (eg: OIDC)

Reply:
The changes are already in the README-setup.md file.

I have now made the corrections as from your indications, please refer to commit:
https://github.com/italia/Satosa-Saml2Spid/pull/104/commits/35bce3de32936895e60eb7d9f518362f8767c22d